examples/v2/red_flags.py

```diff
@@ -1,23 +1,23 @@
-import config
+from infermedica_api.connectors.v2.models import Diagnosis
+from .. import config
 
 config.setup_examples()
 import infermedica_api
 
+if __name__ == "__main__":
+    api: infermedica_api.ModelAPIv2Connector = infermedica_api.get_api("v2")
 
-if __name__ == '__main__':
-    api = infermedica_api.get_api()
-
-    # Prepare diagnosis request it need to have sufficient amount of evidences
+    # Prepare diagnosis request it need to have sufficient amount of evidence
     # The most appropriate way to get a request way for red_flags method is to
     # use the one which has been created while interacting with diagnosis.
     # For the example purpose a new one is created.
-    request = infermedica_api.Diagnosis(sex='female', age=35)
+    request = Diagnosis(sex="female", age=35)
 
-    request.add_symptom('s_10', 'present')
-    request.add_symptom('s_608', 'present')
+    request.add_symptom("s_10", "present")
+    request.add_symptom("s_608", "present")
```

Comment:
This is minor issue, but is there any reason why in the whole project you implemented single quotes instead of double quotes?

Reply:
Now I see `black`...

Reply:
Yes, previously the quotation types were mixed ;/